libs/utils/env.py

```diff
@@ -761,6 +761,18 @@ def _load_em(self, board):
             return None
         return board.json['nrg_model']
 
+    def _load_peripherals(self, board):
+        peripherals_path = os.path.join(basepath,
```

Comment:
This json loading code is pretty redundant. Can we abstract it better? It seems `_load_board` and `_load_em` both have similar code.

Reply:
done.